src/fragments/lib/in-app-messaging/js/getting-started.mdx

```diff
@@ -0,0 +1,125 @@
+Make sure you have completed the below steps:
+
+- [Completed the pre-requisite steps.](/lib/in-app-messaging/prerequisites)
+- [Create a campaign from the pinpoint console.](/lib/in-app-messaging/create-campaigns)
+
+## Integrate your application
+
+In your application directory, you should first install the necessary dependencies. Although Amplify In-App Messaging can be used as a standalone JavaScript library, the only UI integration currently available is only for _React Native_. The integrations steps here assume a React Native application as the implementation target.
+
+```bash
+npm install -E aws-amplify@in-app-messaging aws-amplify-react-native@in-app-messaging amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker react-native-get-random-values react-native-url-polyfill
+```
+
+You will also need to install pod dependencies for `iOS`. Navigate the the ios directory in your project and run the install command.
```

Comment:
Navigate **to** the **iOS** directory in your project and run the install command.